projects/1.16/assets/tea-kettle/tea_kettle/lang/zh_cn.json

```
@@ -15,5 +15,30 @@
   "item.tea_kettle.black_tea_leaf": "红茶叶",
   "effect.tea_kettle.caffeine": "咖啡因",
   "effect.tea_kettle.zen": "禅定",
-  "block.minecraft.bed.caffeine": "过度的咖啡因摄入使你无法安睡"
-}
\ No newline at end of file
+  "block.minecraft.bed.caffeine": "过度的咖啡因摄入使你无法安睡",
+  "block.tea_kettle.empty_kettle": "烧水壶",
+	"block.tea_kettle.water_kettle": "装满水的壶",
+	"block.tea_kettle.water_kettle_hot": "沸腾的烧水壶",
+	"block.tea_kettle.milk_kettle": "装满奶的壶",
+	"block.tea_kettle.milk_kettle_hot": "出沫的壶",
+	"block.tea_kettle.campfire_and_kettle": "营火和壶",
+	"block.tea_kettle.soul_campfire_and_kettle": "灵魂营火和壶",
+	"block.tea_kettle.green_latte": "一杯绿色拿铁",
+	"block.tea_kettle.white_latte": "一杯白色拿铁",
+	"block.tea_kettle.oolong_latte": "一杯乌龙拿铁",
+	"block.tea_kettle.black_latte": "一杯红石拿铁",

```

```suggestion
	"block.tea_kettle.black_latte": "一杯红茶拿铁",

```
貌似是红茶吧……原文是个`Black Latte`